package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -210,23 +253,150 @@ class WatsonCrickDist(NucPairDist):
     """
 
     def __init__(self, strand1: List[Residue], strand2: List[Residue],
-                 n1_name: str = 'N1', n3_name: str = "N3",
-                 g_name: str = 'G', a_name: str = 'A', u_name: str = 'U',
-                 t_name: str = 'T', c_name: str = 'C',
+                 nitrogen1_name: str = 'N1', nitrogen3_name: str = "N3",
+                 guanine_name: str = 'G', adenine_name: str = 'A', uracil_name: str = 'U',
+                 tymine_name: str = 'T', cytosine_name: str = 'C',
                  **kwargs) -> None:
-        sel1: List[mda.AtomGroup] = []
-        sel2: List[mda.AtomGroup] = []
-        strand = zip(strand1, strand2)
 
-        for s in strand:
-            if s[0].resname[0] in [c_name, t_name, u_name]:
-                a1, a2 = n3_name, n1_name
-            elif s[0].resname[0] in [a_name, g_name]:
-                a1, a2 = n1_name, n3_name
-            else:
-                raise ValueError(f"{s} are not valid nucleic acids")
+        selections: Tuple[List[mda.AtomGroup], List[mda.AtomGroup]] = self.select_strand_atoms(
+            strand1, strand2, nitrogen1_name, nitrogen3_name, 
+            guanine_name=guanine_name, adenine_name=adenine_name,
+            tymine_name=tymine_name, uracil_name=uracil_name, cytosine_name=cytosine_name
+        )
+
+        super(WatsonCrickDist, self).__init__(selections[0], selections[1], **kwargs)
+
+
+class MinorPairDist(NucPairDist):
+    r"""Minor-Pair basepair distance for selected residues over a trajectory.
+
+    Takes two lists of :class:`~MDAnalysis.core.groups.Residue` objects and
+    calculates the Watson-Crick distance between them over the trajectory.
```

Comment:
How is `MinorPairDist` different from `WatsonCrickDist`? The docs do not say. Please add details to the docs to describe the difference.

Reply:
Start by adding docs from orginal functions https://docs.mdanalysis.org/stable/documentation_pages/analysis/nuclinfo.html#MDAnalysis.analysis.nuclinfo.minor_pair

Reply:
Explicitly state what distances are calculated.